Library/Formula/libtorrent.rb

```diff
@@ -3,15 +3,20 @@ class Libtorrent < Formula
   homepage "https://github.com/rakshasa/libtorrent"
   # Both homepage and primary url have been down since at least ~April 2015
   # https://github.com/rakshasa/rtorrent/issues/295
-  url "https://mirrors.ocf.berkeley.edu/debian/pool/main/libt/libtorrent/libtorrent_0.13.4.orig.tar.gz"
-  mirror "https://mirrors.kernel.org/debian/pool/main/libt/libtorrent/libtorrent_0.13.4.orig.tar.gz"
-  sha256 "74a193d0e91a26f9471c12424596e03b82413d0dd0e1c8d4d7dad25a01cc60e5"
+  url "http://rtorrent.net/downloads/libtorrent-0.13.6.tar.gz"
+  mirror "https://github.com/rakshasa/libtorrent/archive/0.13.6.tar.gz"
```

Comment:
Can you check the `mirror` has the same checksum as the `url` here? Last time we couldn't do this because they differed.